extensions/amp-consent/0.1/cmps.js

```diff
@@ -41,6 +41,12 @@ CMP_CONFIG['didomi'] = {
   'promptUISrc': 'https://sdk-amp.privacy-center.org/loader.html',
 };
 
+CMP_CONFIG['sirdata'] = {
+  'consentInstanceId': 'sirdata',
+  'checkConsentHref': 'https://sddan.mgr.consensu.org/api/v1/public/amp/check',
+  'promptUISrc': 'http://ui.sddan.mgr.consensu.org/amp.html',
```

Comment:
AMP only supports https protocol. Can we switch to that instead? Thanks.

Reply:
Since AMP only supports `https` protocol, having the default url to be `http` won't work here. You'll need to change the default value and maybe also communicate the message to your client that they can not override the protocol to `http`. Thanks.